### ISSUE TITLE
File Explorer items after latest changes in Obsidian

### ISSUE THREAD

**Issue by iHeadway** [Akifyss/obsidian-border#83]
Now the background highlight is displayed in full file explorer width not item as before.
![bR54bFn 1](https://github.com/Akifyss/obsidian-border/assets/30590849/490ae53a-1193-459c-bf1e-96a49363ae43)

**Comment by Akifyss** [comment]
Yeah, it's a new [feature](https://forum.obsidian.md/t/obsidian-release-v1-3-1-insider-build/59810#improvements-1) of Obsidian and I won't be modifying it at the moment.

**Comment by Akifyss** [comment]
Yeah, I just checked that too, and it looks like it does.😣
